common/ai_budget/00_unity_budget.txt

```diff
@@ -0,0 +1,180 @@
+## Categories used in this file must have use_for_ai_budget = yes defined in economic_categories ##
+## It is possible to have multiple entries with the same category and resource ##
+
+#################
+## Expenditure ##
+#################
+unity_expenditure_edicts = {
+	resource = unity
+	type = expenditure
+	category = edicts
+
+	potential = {
+		always = yes
+	}
+
+	fraction = {
+		weight = 1
+	}
+}
+
+unity_expenditure_traditions = {
+	resource = unity
+	type = expenditure
+	category = traditions
+
+	potential = {
+		has_unlocked_all_traditions = no
+	}
+
+	fraction = {
+		weight = 0.8
+	}
+}
+
+unity_expenditure_megastructures = {
+	resource = unity
+	type = expenditure
+	category = megastructures
+
+	potential = {
+		is_at_war = no
+		can_build_unity_megastructures = yes
```

Comment:
Geth sphere might need to be added here, now that the AI is tech rushing. We probably want to wait until scripted triggers review though.